### DISCUSSION TITLE
SignedApk

### DISCUSSION THREAD

**Discussion by ghost** [AndroidIDEOfficial/AndroidIDE#785] [Q&A]
Hello, I was trying to run jarsigner in terminal to create a key to sign my app, everything was fine, but when I run pkg install apksigner to sign the app, terminal gives an error of type E: Unable to locate package apksigner My question is is it currently possible to sign my app with my custom key using the AndroidIDE terminal?

**Comment by PriNova** [comment]
The `apksigner` is part of the android-sdk. To use the build commands you need to add it to your `PATH` environment. If you build a release variant and point your keystore file to the location, then AndroidIDE will automatically use the keystore file for signing.
